docs/docs/Workspace/workspace-overview.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Workspace Concepts
+sidebar_position: 1
+slug: /workspace-overview
+---
+
+The **Workspace** is where you create AI applications by connecting and running components in flows.
+
+The workspace controls allow you to adjust your view and lock your flows in place.
+
+## Components
+
+A **component** is a single building block within a flow and consists of inputs, outputs, and parameters that define its functionality.
+
+To add a component to your flow, drag it from the sidebar onto the workspace.
+
+To connect components, drag a line from the output handle (⚪) of one component to the input handle of another.
+
+For more information, see [How to build flows with components](/components-overview).
+
+## Playground
+
+The **Playground** executes the current flow in the workspace.
+
+Chat with your flow, view inputs and outputs, and modify your AI's memories to tune your responses in real time.
+
+Any input or output component can be opened in the **Playground** and tested in real time.
+
+For more information, see the [Playground](/workspace-playground).
+
+## API
+
+The **API** pane provides code templates to integrate your flows into external applications.
+
+For more information, see the [API](/workspace-api).
+
+## Collections, Folders, and Projects
+
+The **My Collection** page displays all the flows and components you've created in the Langflow Workspace.
+
+Your **collection** is the top-level categorization for all of your projects.
+
+### My Projects
+
+**My Projects** is the default folder where all new projects and components are initially stored.
+
+* To create a new folder, click 📁 **New Folder**.
+
+To rename a folder, double-click the folder name.
+
+To download a folder, click 📥 **Download**.
+
+To upload a folder, click 📤 **Upload**.
```

Comment:
This is stating the obvious. Can you provide any info that's not obvious, for example is there a maximum number of folders, can you delete the default folder, etc.